main.py

```diff
@@ -0,0 +1,29 @@
+#%%
+import cloudinary
+import cloudinary.uploader
+from cloudinary.utils import cloudinary_url
+
+# Configuration       
+cloudinary.config( 
+    cloud_name = "dfxefh712", 
+    api_key = "936813184334185", 
+    api_secret = "w6SdsGJqrm34jlKZuzvKQqHES4o", # Click 'View API Keys' above to copy your API secret
```

Comment:
@someshfengde please remove your secret from this file, since everyone can see it and it is already compromised.

You can disable existing and generate a new one here:
https://console.cloudinary.com/settings/api-keys

Reply:
Sorry removed the main.py file and disabled the secret

Reply:
Thanks for informing